### PR TITLE
FR47180 Holistic instrument tracking folders - outlier level setting ability

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
This PR provides support for displaying lower bound and upper bound outlier level setting with related PR.

#### Related Pull Requests
* https://github.com/LabKey/targetedms/pull/822

### CODE REVIEW

File: core/webapp/vis/src/plot.js [hunk 1]
```diff
@@ -2175,41 +2206,71 @@ boxPlot.render();
...
+                            geom: new LABKEY.vis.Geom.ErrorBar({size: 1, color: 'red', dashed: true, width: barWidth, topOnly: true}),
+                            data: meanStdDevData,
+                            aes: {
+                                error: function (row) {
+                                    return row[config.properties.stdDev] * config.properties.lowerBound;
+                                },
+                                yLeft: config.properties.mean
+                            }
+                        }));
+                        // Are we OK suppressing the 1x and 2x bars for all use cases?
```
Comment (labkey-jeckels) [inline core/webapp/vis/src/plot.js]: I think we decided that we want to retain the 1x and 2x bars for non-Panorama use cases. Is that happening already or do we need to handle that? @cnathe can provide non-Panorama use cases if that's helpful.

**Review by labkey-jeckels — APPROVED** [review]
Question on the +/- 1 and 2 SD lines. Please either remove the comment or make sure that they are retained for non-Panorama use cases.

File: core/webapp/vis/src/plot.js (hunk 1 above)
Comment (ankurjuneja) [reply core/webapp/vis/src/plot.js]: I retained those with a checkbox option but 2nd item in feature verification suggested to remove those. CC: @wendyinnis

Comment (labkey-jeckels) [reply core/webapp/vis/src/plot.js]: Right - we don't want to show them in Panorama. But we do want to show them in other places that use the Levey-Jennings plot but always use +/- 3 SDs as the cutoffs

**Pull request merged**